src/app/layout.tsx

```diff
@@ -14,8 +13,20 @@ interface RootLayoutProps {
 
 export const metadata: Metadata = {
   title: "Guildhall",
+  applicationName: "Guildhall",
   description:
     "Automated membership management for the platforms your community already uses.",
+  icons: {
+    icon: "guild-icon.png",
+  },
+}
+
+export const viewport: Viewport = {
+  themeColor: [
+    { media: "(prefers-color-scheme: dark)", color: "#000000" },
```

Comment:
I think we could use the current dark gray background color here and in the `manifest.ts` file too (`#27272a`)